spec/validate_url_spec.rb

```diff
@@ -271,4 +271,20 @@
       expect(@user.errors[:homepage]).to eq(["wrong"])
     end
   end
+
+  context "with SNS" do
+    before do
+      @user = UserWithArAndSns.new
+    end
+
+    it "should not allow a valid sns uri" do
```

Comment:
Recommend adding more context in the test description here and on line 285 (E.g. which part of uri is broken)